docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -281,16 +286,14 @@ Release
 
 #. Create and activate the Flocker release virtual environment:
 
-   .. note:: The following instructions use `virtualenvwrapper`_ but you can use `virtualenv`_ directly if you prefer.
-
-   .. prompt:: bash $
+   .. prompt:: bash [vagrant@localhost]$
```

Comment:
Reviewer: Note that this is not ideal. We noted that the real vagrant prompt includes the last part of the current path, but thought that it is not worth emulating this. This approach makes it clear that you are in the vagrant box.